pennylane/circuit_drawer/representation_resolver.py

```diff
@@ -319,67 +319,78 @@ def operator_representation(self, op, wire):
 
             return (" " + self.charset.OTIMES + " ").join(constituent_representations)
 
-        name = op.name
+        representation = ""
+        base_name = getattr(op, "base_name", op.name)
+        name = base_name
```

Comment:
How come base_name and name refer to the same value? Better to just use a single variable?

Reply:
No, `base_name` is `op.base_name` if it exists (the method should work for both Observables and Operations), otherwise it is `op.name`. In the rest of the code, `name` might be resolved further (i.e. `CNOT` goes to `X` on the controlled wire) while `base_name` is used for resolving stuff (e.g. seeing which wires are control wires)